src/mmw/js/src/account/views.js

```diff
@@ -125,14 +183,17 @@ var AccountContainerView = Marionette.LayoutView.extend({
 
     initialize: function() {
         this.tokenModel = new models.ApiTokenModel();
+        this.profileModel = new userModels.UserProfileModel(App.user.attributes.profile);
```

Comment:
Prefer `App.user.get('profile')` rather than directly accessing the `attributes`.

Reply:
Addressed in fixup 2259d26